contracts/Comptroller.sol

```diff
@@ -1019,6 +1030,40 @@ contract Comptroller is ComptrollerV3Storage, ComptrollerInterface, ComptrollerE
             require(allMarkets[i] != CToken(cToken), "market already added");
         }
         allMarkets.push(CToken(cToken));
+
+        // Initially, set borrow limit to unlimited. This should be set appropriately in the same governance call immediatly after adding the market.
+        _setMarketBorrowLimitInternal(CToken(cToken),uint(-1));
```

Comment:
I think if we use `0` as "unlimited", then we don't need this. Plus even in this code, don't we set this in the constructor.

Reply:
Well, with 0 as unlimited, yes. This is for when new markets are added beyond the current ones, they default to infinity too.